Doc/library/sqlite3.rst

```diff
@@ -25,6 +25,15 @@ The sqlite3 module was written by Gerhard Häring.  It provides an SQL interface
 compliant with the DB-API 2.0 specification described by :pep:`249`, and
 requires SQLite 3.7.15 or newer.
 
+This document includes four main sections:
+
+* :ref:`sqlite3-tutorial`: teaches how to use the sqlite3 module
+* :ref:`sqlite3-reference`: describes the classes and methods defined in the
+  module
+* :ref:`sqlite3-howtos`: explains how to solve specific problems
```

Comment:
```suggestion
* :ref:`sqlite3-howtos` explains how to handle specific tasks
```

* Tweak wording to mention positive (and more general) "tasks" rather than negative (and not always encompassing) "problems"
* Frame as a prose sentence

Reply:
The singular/plural agreement sounds kinda weird, even if it is technically correct (referring to the singular section rather than the plural guides). I considered proposing referring to the underlying item rather than the section name to avoid this, but decided not to for now. We _could_ reference the guides rather than the section here, but that wouldn't be consistent with the others (especially since Concepts is also a plural word, even if the object it references, a section, is singular, whereas the "guides" are actually multiple distinct objects, just under one section)

Reply:
Or maybe I'm overthinking this, heh

Reply:
> We could reference the guides rather than the section here [...]

I don't think that is needed, as you've got the ToC menu to your immediate left ;)

Reply:
I do like the proposed wording, though.

Reply:
> I don't think that is needed, as you've got the ToC menu to your immediate left ;)

Sorry, I didn't mean to imply actually enumerating the guides or anything, but rather the semantic nuance of referring to "How-to guides" the _section_ (singular) versus "How-to guides" the _guides_ (plural)

Reply:
Ah, got it :)

Reply:
I'm marking this as resolved. Let me know if you disagree :)